src/pages/younghyunKim/components/WritingComment.scss

```
@@ -0,0 +1,25 @@
+.commentDiv {
```

```suggestion
.writingComment {
```
셀렉터 이름에 태그 이름이 들어가는거 어색한거 같아요